packages/vuetify/src/components/VAutocomplete/VAutocomplete.tsx

```diff
@@ -520,6 +520,7 @@ export const VAutocomplete = genericComponent<new <
                                       modelValue={ isSelected }
                                       ripple={ false }
                                       tabindex="-1"
+                                      aria-label={ isSelected ? 'checked' : 'not checked' }
```

Comment:
Does this need to use i18n?

Reply:
Probably makes sense. I tried to add i18n (added English for most languages, I hope that would be okay).